vnext/local-cli/generator-windows/templates/cpp/proj/MyApp.vcxproj

```diff
@@ -20,6 +20,9 @@
     <PackageCertificatePassword>password</PackageCertificatePassword>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
+  <PropertyGroup Label="ReactNativeWindowsNodeProps">
```

Comment:
>Node [](start = 42, length = 4)

I wonder why we need the 'Node' word here and below.

Reply:
We don't, removed it.